tests_integration/test_cp.py

```diff
@@ -19,7 +19,7 @@ def test_cp(runner, snowflake_session):
             "create",
             "--name",
             cp_name,
-            "--num",
+            "--num-instances",
```

Comment:
Looks like the integration test is not enabled. Do we have any plan to run spcs integration test? cc: @sfc-gh-turbaszek

Reply:
Tomasz enabled them with his recent merged pr: https://github.com/Snowflake-Labs/snowcli/pull/684